### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by aschmahmann**
I'm not sure why this repo had master utilize non-released repos, but this updates them to released versions.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,17 +6,16 @@ require (
...
+	github.com/ipfs/go-merkledag v0.5.1
 	github.com/ipfs/go-unixfs v0.2.4
-	github.com/ipld/go-car/v2 v2.1.0
+	github.com/ipld/go-car/v2 v2.1.1
 	github.com/ipld/go-codec-dagpb v1.3.0
-	github.com/ipld/go-ipld-prime v0.12.4-0.20211014180653-3ba656a3bc6b
-	github.com/multiformats/go-multicodec v0.3.1-0.20210902112759-1539a079fd61
-	github.com/multiformats/go-multihash v0.0.17-0.20211012170226-654b06d7912a
+	github.com/ipld/go-ipld-prime v0.14.3
+	github.com/multiformats/go-multicodec v0.3.1-0.20211210143421-a526f306ed2c
```
Comment (aschmahmann) [inline go.mod:17]: Seems like we're waiting on a release of go-multicodec. Anything we're waiting on @mvdan?

**Review by willscott — APPROVED** [review]
:+1: maybe worth waiting for a multicodec tag as you note

File: go.mod (hunk 1 above)
Comment (mvdan) [reply go.mod:17]: https://github.com/multiformats/go-multicodec/releases/tag/v0.4.0

Comment (mvdan) [reply go.mod:17]: ^ bump in case you didn't see it during the holidays :)

**Pull request merged**